tools/formatting/format-check.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python
```

Comment:
On some systems default `python` could `python3` on another it could be `python2`. To be sure in such cases is better to specify at least major version of python.

Reply:
This should work with both `python2` and `python3` and that's why I'm not restricting.